src/components/Filter/Filter.js

```diff
@@ -41,46 +61,122 @@ const Filter = () => {
     }
   }, [filter.type]);
 
+  const count = Math.ceil(filterResult.length / perPage);
+  console.log("Total filtered recipes", filterResult.length);
+  console.log("Total pages", count);
+  const pageData = ReusablePagination(filterResult, perPage);
+
+  const handleChange = (event, p) => {
+    setPage(p);
+    pageData.jump(p);
+  };
+
   return (
     <Container>
+      <Typography
+        className="showElement"
+        align="center"
+        variant="h2"
+        mt={9}
+        mb={5}
+      >
+        Search results for {filter.search}
+      </Typography>
       <Grid
```

Comment:
The filter part is  not responsive.
<img width="523" alt="Screen Shot 2022-12-05 at 7 47 35 PM" src="https://user-images.githubusercontent.com/88150284/205811662-446d73e9-e710-460c-8e7e-37dcd6545ac7.png">